modules/developers-guide/examples/javascript/node-full-script.mjs

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env zx
```

Comment:
Do we need to call out that this example uses zx? FWIW I think the script as is could just be the `index.js` of a Node application and run without issue, so we could potentially just remove this line entirely.

Reply:
I added a note in the full script: 

// This script uses zx - if you wish to rename to index.js and use in a node application
// remove the line above.